packages/dm-core-plugins/src/form/components/AttributeList.tsx

```diff
@@ -43,18 +34,11 @@ export const AttributeList = (props: {
           namePath={`${prefix}${attribute.name}`}
           attribute={attribute}
           uiAttribute={uiAttribute}
-          readOnly={readOnly}
+          readOnly={config?.readOnly}
         />
       </div>
     )
   })
 
-  return (
-    <Stack spacing={1}>
-      {config?.editToggle && (
-        <Button onClick={toggleHandler}>{readOnly ? 'Edit' : 'View'}</Button>
-      )}
-      {attributeFields}
-    </Stack>
-  )
+  return <Stack spacing={1}>{attributeFields}</Stack>
```

Comment:
You don't really need the `Stack` either
```suggestion
  return <>{attributeFields}</>
```